opensrp-giz-malawi/src/main/java/org/smartregister/giz/recievers/SyncStatusReciever.java

```diff
@@ -0,0 +1,40 @@
+package org.smartregister.giz.recievers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.smartregister.domain.FetchStatus;
+import org.smartregister.giz.job.GizReProcessJob;
+
+import java.io.Serializable;
+
+import timber.log.Timber;
+
+import static org.smartregister.receiver.SyncStatusBroadcastReceiver.EXTRA_FETCH_STATUS;
+
+public class SyncStatusReciever  extends BroadcastReceiver {
```

Comment:
Rename this to `GizReprocessSyncStatusReceiver` since this is only being used by the `GizClientReprocessJob`